src/content/chainlink-automation/guides/manage-upkeeps.mdx

```diff
@@ -23,8 +23,8 @@ You must monitor the balance of your Upkeep. If the Upkeep LINK balance drops be
 
 <Aside type="tip" title="ERC-677 Link">
   For funding on Mainnet, you need ERC-677 LINK. Many token bridges give you ERC-20 LINK tokens. Use PegSwap to [convert
-  Chainlink tokens (LINK) to be ERC-677 compatible](https://pegswap.chain.link/). To fund on a supported testnet, get
-  [LINK](/resources/link-token-contracts) from [faucets.chain.link](https://faucets.chain.link/).
+  Chainlink tokens (LINK) to be ERC-677 compatible](https://pegswap.chain.link/). On testnets, use
+  [faucets.chain.link](https://faucets.chain.link/) to get testnet LINK.
```

Comment:
```suggestion
  Chainlink tokens (LINK) to be ERC-677 compatible](https://pegswap.chain.link/). Use
  [faucets.chain.link](https://faucets.chain.link/) to get testnet LINK.
```

Reply:
Nit: Removing "On testnets" to make the sentence sound a little less redundant; if you agree, apply same change throughout the rest of the PR